crates/orchestrator/src/data_storage/aws_s3/mod.rs

```diff
@@ -48,7 +48,15 @@ impl DataStorage for AWSS3 {
     async fn get_data(&self, key: &str) -> Result<Bytes> {
         let response = self.client.get_object().bucket(&self.bucket).key(key).send().await?;
         let data_stream = response.body.collect().await.expect("Failed to convert body into AggregatedBytes.");
+        tracing::debug!("DataStorage: Collected response body into data stream from {}, key={}", self.bucket, key);
         let data_bytes = data_stream.into_bytes();
+        tracing::info!(
+            log_type = "DataStorage",
+            category = "data_storage_call",
```

Comment:
what does log_type and category mean? is this meant to be added on the metrics? also, maybe you meant to make this trace and show data_bytes?

Reply:
Adding fields in logs gives otel / signoz ability to track various aspects of code run using logs,
as I am tracking a block_no in all the jobs, so I can filter by that block number to see the logs that are being produced specifically for that and then analyse.

Reply:
data_bytes doesn't impl `to_string` i have logged it's len for now, if that doesn't help we can later look into how to add to_string to it.

Reply:
sounds good!